woocommerce/woocommerce-ios
# [IAP] Parse data for Essential plan benefits

Part of: #9967 


## Description


When a merchant taps the Upgrade now link in the banner shown on Free Trial Woo Express stores, we will show them a page prompting them to upgrade via in app purchase. On this page, we will show plan details, including the benefits of the Woo Essential Monthly plan, which is the first we'll make available.

This PR covers parsing the benefits info added in #9933 into a model object that we can use in the UI

## Testing instructions


No user-visible changes. Unit tests should pass

1. Launch the app with a Woo Express trial store
2. Tap Upgrade Now
3. Observe that `Default plan details loaded: Woo Express Essential Monthly` is logged to the console. 
4. Optionally repeat with a breakpoint on this line, and inspect the plan details – see that the data, including colours, are correctly unwrapped to the model


---
